OBJECT-SERVER.md

```diff
@@ -0,0 +1,27 @@
+# Using Realm Object Server on Android
+
+
+1) Add the following to the top-level `build.gradle` file:
+
+
+```groovy
+buildscript {
+repositories {
+jcenter()
+}
+dependencies {
+classpath "io.realm:realm-gradle-plugin:2.0.0"
```

Comment:
Well, `classpath "io.realm:realm-gradle-plugin:<version>` would also make sense.

That's how the `-SNAPSHOT` is shown as an example.